pages.fr/common/antibody.md

```diff
@@ -0,0 +1,16 @@
+# antibody
+
+> Le manager de module shell "le plus rapide".
```

Comment:
```suggestion
> Le gestionnaire de module shell "le plus rapide".
```

Reply:
nice catch, i'll be more careful next time with `manager` :p